src/public/CoinGecko.json

```diff
@@ -0,0 +1,2321 @@
+{
+  "name": "CoinGecko",
+  "timestamp": "2024-04-17T12:19:50.841Z",
+  "version": {
+    "major": 0,
+    "minor": 0,
+    "patch": 1
+  },
```

Comment:
Is this supposed to be the new version of what we have in the UI right now? https://tokenlists.org/token-list?url=https://files.cow.fi/tokens/CoinGecko.json

If that's the case, the version should be 0.3.0

Reply:
Fixed